tests/system/runtimes/test_databricks.py

```diff
@@ -161,14 +166,37 @@ def test_kwargs_from_file(self):
             handler="func",
             auto_build=True,
             project="databricks-proj",
-            params={
-                "task_parameters": {"timeout_minutes": 15},
-                "param1": "value1",
-                "param2": "value2",
-            },
+            params=PARAMS,
         )
         assert run.status.state == "completed"
         assert (
             run.status.results["databricks_runtime_task"]["logs"]
             == "{'param1': 'value1', 'param2': 'value2'}\n"
         )
+
+    @pytest.mark.parametrize(
+        "handler, databricks_code",
+        [
+            # ("print_kwargs", PRINT_KWARGS),
+            (
+                "",
+                PRINT_KWARGS.replace("print_kwargs", "handler"),
```

Comment:
Better not put the actual code in the parameter but just a flag as to whether to replace or not. The long string makes the test results log a lot less readable.